src/app/components/Tab/Tab.tsx

```diff
@@ -0,0 +1,41 @@
+import { Tab as HeadlessUiTab } from "@headlessui/react";
+import { classNames } from "~/app/utils/index";
+
+type Props = React.ButtonHTMLAttributes<HTMLButtonElement> & {
+  active?: boolean;
+  icon?: React.ReactNode;
+  leftTab?: boolean;
```

Comment:
leftTab and rightTab still need to be removed.

Reply:
I also removed active property and used `selected` props from the headlessTab className to have the same effect.